vault/identity_store_oidc.go

```diff
@@ -133,7 +137,7 @@ func oidcPaths(i *IdentityStore) []*framework.Path {
 
 				"algorithm": {
 					Type:        framework.TypeString,
-					Description: "Signing algorithm to use. This will default to RS256, and is currently the only allowed value.",
+					Description: "Signing algorithm to use. This will default to RS256.",
 					Default:     "RS256",
```

Comment:
Would there be any reason to consider a different default?

Reply:
RS256 is the only (asymmetric) method specified as required in the JWT RFC. OIDC is a bit different but it's a reasonable default for now.